jwst/datamodels/multislit.py

```diff
@@ -13,7 +13,7 @@ class MultiSlitModel(JwstDataModel):
     This model has a special member `slits` that can be used to
     deal with an entire slit at a time.  It behaves like a list::
 
-       >>> from .slit import SlitModel
+       >>> from jwst.datamodels.slit import SlitModel
```

Comment:
This would be better as

```
>>> from jwst.datamodels import SlitModel
```

as that's the recommended canonical import.

Reply:
of course! I must have missed some docstrings